Unable to add package

I am not sure if this is the correct way to ask for help so apologies in advance if not.  I have tried adding the package to my project put continuously receive the following error which I have no idea of how to resolve.

Any help appreciated.


Showing Recent Messages
invalidManifestFormat("/var/folders/st/zlvsrg1d3mg3ctxx6j8fzfmc0000gn/T/TemporaryFile.M9cbby.swift:23:17: error: expected \',\' separator\n            path: \"SwiftYNAB/SwiftYNABTests\"\n                ^\n                ,\n/var/folders/st/zlvsrg1d3mg3ctxx6j8fzfmc0000gn/T/TemporaryFile.M9cbby.swift:23:17: error: expected expression in container literal\n            path: \"SwiftYNAB/SwiftYNABTests\"\n                ^\n/var/folders/st/zlvsrg1d3mg3ctxx6j8fzfmc0000gn/T/TemporaryFile.M9cbby.swift:23:13: error: cannot find \'path\' in scope\n            path: \"SwiftYNAB/SwiftYNABTests\"\n            ^~~~", diagnosticFile: Optional(<AbsolutePath:"/Users/NameOfMyMac/Library/Developer/Xcode/DerivedData/YNABSwiftTest-bupfkjycbrorbrhaxbvtsuhngojm/SourcePackages/ManifestLoading/swiftynab.dia">)) in https://github.com/andrebocchini/swiftynab

--- a/Package.swift
+++ b/Package.swift
@@ -19,8 +19,8 @@ let package = Package(
         ),
         .testTarget(
             name: "SwiftYNABTests",
-            dependencies: ["SwiftYNAB"]),
-            path: "SwiftYNAB/SwiftYNABTests"
+            dependencies: ["SwiftYNAB"],
+            path: "SwiftYNAB/SwiftYNABTests")
     ],
     swiftLanguageVersions: [.v5]
 )

